test/ctia/test_helpers/core.clj

```diff
@@ -65,6 +65,7 @@
          "ctia.metrics.riemann.enabled"               false
          "ctia.metrics.console.enabled"               false
          "ctia.metrics.jmx.enabled"                   false
+         "ctia.store.es.identity.refresh"             "true"
```

Comment:
It forces the identity index refresh. It fixes a race condition when we set a new identity with `set-capabilities` in test but the test can start before the index is refreshed and make calls fails with `:missing_capability` error.
I discovered it with a test that failed only in gh and not on my local environment.